file-formats/sito/sito-v1.json

```diff
@@ -0,0 +1,893 @@
+                      "type": "string",
+                      "enum": [
+                        "allApps"
+                      ],
+                      "enumDescriptions": [
+                        "All Apps"
+                      ]
+                    },
+                    "sitnobjstrucvhtype": {
+                      "title": "Type of Value Help",
+                      "description": "Type of Situation Value Help",
+                      "type": "string",
+                      "enum": [
+                        "service"
+                      ],
+                      "enumDescriptions": [
+                        "Service"
+                      ]
+                    },
+                    "sitnobjstrucvhscope": {
+                      "title": "Source",
+                      "description": "Object Source: SAP or Customer",
+                      "type": "string",
+                      "enum": [
+                        "sap",
+                        "customer"
+                      ],
+                      "enumDescriptions": [
+                        "SAP",
+                        "Customer"
+                      ]
+                    },
+                    "sitnobjstrucfieldname": {
+                      "title": "Object Field",
+                      "description": "Object Field",
+                      "type": "string",
+                      "maxLength": 30
+                    }
+                  },
+                  "additionalProperties": false
+                }
+              },
+              "sit2OsVhMapS": {
+                "title": "List of SIT2_OS_VH_MAP_S",
+                "description": "List of SIT2_OS_VH_MAP_S",
+                "type": "array",
+                "items": {
+                  "title": "SIT2_OS_VH_MAP_S",
+                  "description": "SIT2_OS_VH_MAP_S",
+                  "type": "object",
+                  "properties": {
+                    "sitnobjstrucvhapptype": {
+                      "title": "App for Value Help",
+                      "description": "App for Situation Value Help",
+                      "type": "string",
+                      "enum": [
+                        "allApps"
+                      ],
+                      "enumDescriptions": [
+                        "All Apps"
+                      ]
+                    },
+                    "sitnobjstrucfieldname": {
+                      "title": "Object Field",
+                      "description": "Object Field",
+                      "type": "string",
+                      "maxLength": 30
+                    },
+                    "sitnobjvhsrventityset": {
+                      "title": "Name",
+                      "description": "Service Builder: Node Name: 40 chars",
+                      "type": "string",
+                      "maxLength": 40
+                    },
+                    "sitnobjvhsrvproperty": {
+                      "title": "Name",
```

Comment:
"sitnobjvhsrvproperty" does not match with "title": "Name". Is this a copy paste mistake?

Reply:
I am not sure whether I understand your question. Do you want to know how you can change the title for one of your fields?

This is documented here: https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#title